action.yml

```
@@ -152,6 +152,38 @@ runs:
         GITHUB_REPOSITORY: ${{ github.repository }}
         GITHUB_SHA: ${{ github.sha }}
       run: |
+        # Work out if release should be marked as the latest
+        # Only do the for stable semver tags
+        # Note = not using "legacy" as GitHub seems to only consider if it's the latest tag in a particular major line
+        # whereas we only want to mark it the latest if it's the latest tag overall
+        MAKE_LATEST="false"
+        if [[ $TAG =~ ([0-9]+)\.([0-9]+)\.([0-9]+) ]]; then
```

```suggestion
        if [[ $TAG =~ ^([0-9]+)\.([0-9]+)\.([0-9]+)$ ]]; then
```
Otherwise pre-stable tags will also match.

---

Updated